src/utilities/get.ts

```diff
@@ -0,0 +1,17 @@
+import {GeneralObject} from '../types';
+
+function get(obj: GeneralObject | undefined, keypath: string) {
```

Comment:
I guess we never use a default with the lodash function, so you didn't a default in your custom function?

Reply:
Yup but it's easy enough to add so I might as well